.config/objectstore.config.php

```diff
@@ -0,0 +1,21 @@
+<?php
+if (getenv('OBJECTSTORE_S3_BUCKET')) {
+  $use_ssl = getenv('OBJECTSTORE_S3_SSL');
+  $use_path = getenv('OBJECTSTORE_S3_USEPATH_STYLE');
+  $CONFIG = array(
+    'objectstore' => array(
+      'class' => '\OC\Files\ObjectStore\S3',
+      'arguments' => array(
+        'bucket' => getenv('OBJECTSTORE_S3_BUCKET'),
+        'key' => getenv('OBJECTSTORE_S3_KEY') ?: '',
+        'secret' => getenv('OBJECTSTORE_S3_SECRET') ?: '',
+        'region' => getenv('OBJECTSTORE_S3_REGION') ?: '',
+        'hostname' => getenv('OBJECTSTORE_S3_HOST') ?: '',
+        'port' => getenv('OBJECTSTORE_S3_PORT') ?: '',
+        'use_ssl' => $use_ssl != null && (strtolower($use_ssl) == 'false' || $use_ssl == false) ? false : true,
```

Comment:
The null check here isn't necessary since `getenv()` will return `false` if the environment variable wasn't set, not `null`. Also can we use the `===` operator to avoid type coercion? https://www.php.net/manual/en/function.getenv

Reply:
Dropped the null check. Good spot. I wanted to keep the type coercion for 0 and 1 (you never know what somebody's logic/script/orchestrator may fill down more easily to this container).

Reply:
Ah, makes sense